.github/workflows/scan.yml

```diff
@@ -0,0 +1,22 @@
+name: scan-images
+
+on:
+  schedule:
+    - cron: "0 12 * * 1"
+
+# Remove all permissions from GITHUB_TOKEN except metadata.
+permissions: {}
+
+jobs:
+  scan:
+    name: Trivy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # tag=v3.1.0
```

Comment:
nit: Is there a way to use an official version tag here rather than a sha?

Reply:
I think we're always using the SHA + tag in a comment for all the workflows

Reply:
SHA is more granular method of pinpointing a specific release of an artifact and is also the recommended way https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses

"Using the commit SHA of a released action version is the safest for stability and security."